daemon/oci_linux.go

```diff
@@ -655,6 +656,29 @@ func (daemon *Daemon) createSpec(c *container.Container) (*specs.Spec, error) {
 	}
 	s.Linux.Resources.OOMScoreAdj = &c.HostConfig.OomScoreAdj
 	s.Linux.Sysctl = c.HostConfig.Sysctls
+
+	p := *s.Linux.CgroupsPath
```

Comment:
there is no need to do all of the below in case CPU RT parameters are not set, or the kernel does not support CPU RT sched.